docs/howto/features/index.md

```diff
@@ -5,6 +5,57 @@ There are several extra features and services that can be enabled on the 2i2c Ju
 These add extra functionality, connect with cloud services, etc.
 See the sections below for more details.
 
+## Layers
+
+### Cloud
+1. Shared/Dedicated Cluster
+2. Dedicated nodepool on a shared cluster (recommended for events)
+
+### Hub User Access
+1. Authentication + authorization
+2. Anonymisation of users ids
+
+### Community customizations
+1. Community specific hub domain
+2. (mandatory) Hub login page template sections
+3. Self-configuration of hub login page from own GitHub repository
+4. Customizations of hub pages
+
+### Data
+1. Shared data directories
+2. Object storage buckets (persistent/scratch)
+  - Cloud Permissions
+    - (default) Buckets accessible from the Hub
+    - Buckets accessible from outside the Hub
+      - Publicly accessible
+      - (GCP only) Google groups based membership
+    - (GCP only) Requestor pays for buckets
+
+### Integrations
+
+1. Allow users to push to GitHub from the Hub (gh-scoped-creds)
+2. Enable `nbgitpuller` for private repos (git-credential-helper)
+3. (dedicated clusters only) Grafana
+4. Authenticated static websites
+5. (default enabled) Configurator
+
+### Performance
+
+1. GPUs
+  - GPU user profile
+2. Dask
+3. KernelCulling
+
+### Elastic Scaffolding
+
+1. ProfileLists
+  - Hardware Profiles
+  - User Image Selectors
+  - Node Sharing
+
+### Knowledge sharing
```

Comment:
### Support Layer

(Here @colliand requests input from @jmunroe on how to break down the `Support Layer` consistent with the granularity on this page. This will likely need to be developed in another issue thread.)